pkg/apis/mongodb/v1/mongodb_types.go

```diff
@@ -68,6 +68,15 @@ type MongoDBSpec struct {
 	AdditionalMongodConfig MongodConfiguration `json:"additionalMongodConfig,omitempty"`
 }
 
+func (m MongoDBSpec) ToJson() []byte {
+	b, err := json.Marshal(m)
+	if err != nil {
```

Comment:
I think we should at least log this error, we won't have any insight into this otherwise